en/docs/integrations/integration-concepts.md

```diff
@@ -1,57 +1,55 @@
+  
 ## Integration
-Integration in Choreo is the process of connecting APIs, events, and streams to function as a single unit that can share information.
-
-## Integration Template
-An integration template in Choreo is a prebuilt sample designed to address a common integration requirement. Choreo provides a comprehensive set of integration templates for you to select and use depending on your requirement.
+Integration in Choreo is a process for connecting APIs, events, and streams to function as a single unit. Choreo provides an integration template which is a prebuilt sample designed to address a common integration requirement. Choreo includes a set of comprehensive integration templates for you to select and use.
 
 ## Trigger
 A trigger in Choreo integrations is an event or an action that can cause an integration to start executing.
 
 Choreo supports the following trigger types:
 
- - **Schedule**: To trigger an integration based on a schedule that you set.
- - **Manual**: To run and test an integration manually.
- - **Calendar**: To trigger an integration based on events in a Google Calendar.
- - **GitHub**: To trigger an integration based on GitHub events and actions.
- - **Salesforce**: To trigger an integration based on topics in a Salesforce account. 
+ - **Schedule**: Triggers execution according to a given schedule.
+ - **Manual**: Triggers execution manually by clicking 'Run & Test'.
+ - **Calendar**: Triggers execution based on events in a Google Calendar.
+ - **GitHub**: Triggers execution based on GitHub events and actions.
+ - **Salesforce**: Triggers execution based on Salesforce events. 
 
 ## API Call
-An API call in Choreo integrations allows you to connect to and interact with an external service through your integration. You can use an API call to interact with generic protocol-based APIs and application-specific APIs.
+An API call in Choreo allows you to connect to an external service through your Choreo integration. You can use an API call to interact with generic protocol-based APIs and application-specific APIs.
 
 Following are the connections you can use to interact with generic protocol-based APIs:
   
- - **HTTP**: To interact with the HTTP API and communicate with an external endpoint via the HTTP protocol.   
- - **SMTP**: To send emails via the SMTP protocol.   
- - **POP3**: To receive emails from a POP3 email server.  
- - **IMAP**: To receive emails from any email server that supports the IMAP protocol. 
+ - **HTTP**: Communicates with external endpoints over HTTP.   
+ - **SMTP**: Performs email related operations over SMTP. 
+ - **POP3**: Retrieve emails over POP3. 
+ - **IMAP**: Retrieve emails from a mail server over a TCP/IP connection.
```

Comment:
Shall we say "Retrieves" instead of "Retrieve"

Reply:
Fixed.